tests/ui/specs/DatePicker/Agate-DatePicker-Page.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
```

Comment:
You don't need to use'-' for Page file name.
In the PLAT-126330, there was not.
Agate-PanelPage.js is Better.
In fact, I don't think it's good to prefix of "Agate-".
"-Spec" and "-View" is mandatory to filter on ui-test-utils. but Page is not. just use camelcase.

Reply:
I will change. thanks!